.github/workflows/check.yaml

```
@@ -21,7 +21,7 @@ jobs:
     uses: insightsengineering/r.pkg.template/.github/workflows/audit.yaml@main
   r-cmd:
     name: R CMD Check 🧬
-    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
+    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@38b63dee0b54d8b50dd360b1b40258feb081ed54
```

```suggestion
    uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
```

Just adding a suggestion to block accidental merge with this.

---

But I think you can just use the feature branch here so any updates on the branch will be reflected by simply triggering the workflow here.

---

The problem with the feature branch was that it didn't like having multiple @ (the branch name is `580_module_tests@main`): [`338e058` (#1329)](https://github.com/insightsengineering/teal.modules.clinical/pull/1329/commits/338e0583f102d00ef0a23673130c5b11b3c30578) and it didn't trigger the tests. After that I use the commits to be able to run the checks manually.